reportTemplates/styles/headerText.js

```diff
@@ -0,0 +1,30 @@
+module.exports = `
+  .header-title {
+    color: #D43425;
+    text-transform: uppercase;
+    font-family: Oswald;
+  }
+
+  .header-text-container {
+    page-break-inside: avoid;
+  }
+
+  .text-item {
+    display: -webkit-flex;
+    font-size: 12px;
+    margin-right: 30px;
+  }
+
+  .checkbox-value {
```

Comment:
Unused in HeaderTextBox. Remove

Reply:
Hmm, or recompile the templates since this isn't in the reportTemplateSrc version.